src/screens/SendCoinsScreen.tsx

```diff
@@ -535,8 +535,12 @@ export class SendCoinsScreen extends Component<Props, State> {
     return newAddresses;
   };
 
+  isStateSettedUp = () => this.state !== null && this.state.fromWallet && this.state.fee;
+
   render() {
+    if (!this.isStateSettedUp()) return null;
```

Comment:
the state will never be null. And what if the user sets fee to 0?

Reply:
> the state will never be null

It's not setted up due to that logic
```
 if (wallets.length === 0) {
      Alert.alert('Before creating a transaction, you must first add a Bitcoin wallet.');
      Alert.alert(i18n.send.error.title, i18n.send.error.description);
      props.navigation.goBack();
    } else {
      if (!fromWallet && wallets.length > 0) {
        fromWallet = wallets[0];
        fromAddress = fromWallet.getAddress();
        fromSecret = fromWallet.getSecret();
      }
      this.state = {
        isLoading: false,
        amount: 0,
        wallets,
        showSendMax: false,
        isFeeSelectionModalVisible: false,
        isAdvancedTransactionOptionsVisible: false,
        recipientsScrollIndex: 0,
        fromAddress,
        fromWallet,
        fromSecret,
        addresses,
        memo: '',
        networkTransactionFees: new NetworkTransactionFee(1, 1, 1),
        fee: 1,
        feeSliderValue: 1,
        bip70TransactionExpiration: null,
        renderWalletSelectionButtonHidden: false,
      };
    }
```
the state is null and start rendering even with `props.navigation.goBack();`, and the app shows error

------------------------


> And what if the user sets fee to 0?

I've checked that and it's probably saved as a string because it was working :thinking: But it needs double check.

Reply:
oh right, this is on the master branch...